bin/manage.js

```diff
@@ -82,7 +82,8 @@ function createClient() {
 program
   .version(pkgData.version || 'unknown')
   .description('Perform various management tasks for the Taskcluster AWS Provisioner')
-  .option('-u, --url [url]', 'URL for the API to work against', 'https://taskcluster-aws-provisioner2.herokuapp.com/v1');
+  .option('-u, --url [url]', 'URL for the API to work against', 'http://localhost:5557/v1');
+  //.option('-u, --url [url]', 'URL for the API to work against', 'https://taskcluster-aws-provisioner2.herokuapp.com/v1');
```

Comment:
It was intentional because I do a lot of testing with delete-all command and I really want to make it extra hard to screw up production.  I'm fine with anything that does not make running on the real server the default :)